policy/modules/system/systemd.te

```diff
@@ -1696,6 +1699,12 @@ allow systemd_importd_t self:udp_socket create_socket_perms;
 allow systemd_importd_t self:unix_dgram_socket sendto;
 allow systemd_importd_t systemd_importd_exec_t:file execute_no_trans;
 
+manage_dirs_pattern(systemd_importd_t, systemd_importd_var_lib_t, systemd_importd_var_lib_t)
+manage_files_pattern(systemd_importd_t, systemd_importd_var_lib_t, systemd_importd_var_lib_t)
+files_var_lib_filetrans(systemd_importd_t, systemd_importd_var_lib_t, dir, "confexts")
+files_var_lib_filetrans(systemd_importd_t, systemd_importd_var_lib_t, dir, "extensions")
+files_var_lib_filetrans(systemd_importd_t, systemd_importd_var_lib_t, dir, "portables")
```

Comment:
In Fedora, /var/lib/portables is a part of systemd, so it should not be needed unlike the others.